apps/web/src/server/applications/edit/case/applications-edit-case.controller.js

```diff
@@ -0,0 +1,66 @@
+import {
```

Comment:
this file will contain the pages for editing the case. details, wording and tests are still in progress (see comment above)